pkg/libinit/root_linux.go

```diff
@@ -110,7 +110,7 @@ var (
 		dev{Name: "/dev/port", Mode: syscall.S_IFCHR | 0640, Dev: 0x0104},
 
 		dir{Name: "/dev/pts", Mode: 0777},
-		mount{Source: "devpts", Target: "/dev/pts", FSType: "devpts", Opts: "newinstance,ptmxmode=666,gid=5,mode=620"},
+		mount{Source: "devpts", Target: "/dev/pts", FSType: "devpts", Opts: "ptmxmode=666,gid=5,mode=620"},
```

Comment:
why was newinstance able to come out? 


This may break testramfs and pflask, though I'm not sure I remember correctly.

Reply:
It should definitely be tested to make sure this doesn't break anything.  In my particular use case which (sadly) involves using an RHEL kernel, having "newinstance" does not work as expected.  I was hoping to not have to fork to make it work, but maybe that's the answer.

Reply:
It turns out that "newinstance" was breaking because `/dev/ptmx` was being created as a dev file rather than a symlink to `/dev/pts/ptmx`.  This should be resolved now.